adapters/oidc/js/pom.xml

```diff
@@ -18,6 +18,22 @@
         <js.projectDir>../../../js</js.projectDir>
         <js.adapter.projectDir>${js.projectDir}/libs/keycloak-js</js.adapter.projectDir>
         <js.adapter.distDir>${js.adapter.projectDir}/dist</js.adapter.distDir>
+        <!-- as the frontend-maven-plugin depends on js.projectDir, list all parts with should be ignored from the checksum -->
```

Comment:
This seems rather unmaintainable in the long term as we continue to expand the `js` directory.

Reply:
There is now a second commit which handles this in a more maintainable way: 9f2b8e7a6a8636013d8e5dd38ee962ffce6b8e4d

Reply:
Great, thanks! Consider this one resolved.